docs/api/get-task-ref.md

```diff
@@ -0,0 +1,55 @@
+# Reference:  Get tasks by a property
+## URL
+```
+{base_url}/tasks?{property}={value}
+```
+Gets a list of tasks matching the specified property.
+
+## Required parameters
+
+| Parameter name | Type | Description |
+| ---- | ---- | ---|
+| `property` | String | The property you want to use to get the tasks. See the **Properties** section below.|
+| `value` | String or Number | The value of the property without the quotation marks. Case-sensitive. Partial matches aren't supported. 
+
+## Properties
+
```

Comment:
If the previous section is "required", what are these? Can you explain that here?

Reply:
This section provides the list of properties that can be used in the `property` parameter. I've added an intro to the section. 

I added this section since this was too much detail to add in the `property` row of the parameter. On a second thought, I could just add a link to the `task` resource, which already has these details.